leases/schema/mutations.py

```diff
@@ -209,15 +210,23 @@ def mutate_and_get_payload(cls, root, info, **input):
                 _("Application must be connected to an existing customer first")
             )
 
-        place = get_node_from_global_id(
-            info,
-            input.pop("place_id"),
-            only_type=WinterStoragePlaceNode,
-            nullable=False,
-        )
+        if place_id := input.pop("place_id", None):  # noqa
```

Comment:
Nice to see the "walrus operator" in action! 😄

Reply:
I KNOW! I was excited to finally be able to use it :happy-cat: